lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -162,7 +176,7 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
     private _tagNameControlVisible = false;
     private _existingTags: TagEntry[];
     private _initialExistingTags: TagEntry[];
-    private onDestroy$ = new Subject<void>();
+    private readonly destroyRef = inject(DestroyRef);
```

Comment:
private readonly field is above so can you move this private readonly fields there to group private readonly fields?